release/pipeline/dags/istio_common_dag.py

```diff
@@ -97,22 +102,25 @@ def getBashSettingsTemplate(extra_param_lst=[]):
       template_list.append("export %s={{ settings.%s }}" % (key, key))
 
   template_list.append("""
-                source    "/tmp/gcb_env.sh"
                 # download gcb_env.sh if it exists, otherwise the local copy will be uploaded
-                gsutil -q cp "gs://${CB_GCS_RELEASE_TOOLS_PATH}/gcb_env.sh" "/tmp/gcb_env.sh"
-                source    "/tmp/gcb_env.sh"
-                gsutil -q cp "/tmp/gcb_env.sh" "gs://${CB_GCS_RELEASE_TOOLS_PATH}/"
+                gsutil -q cp "gs://${CB_GCS_RELEASE_TOOLS_PATH}/gcb_env.sh" "${GCB_ENV_FILE}"
+                if [ $? -ne 0 ]; then
+                  gsutil -q cp "${GCB_ENV_FILE}" "gs://${CB_GCS_RELEASE_TOOLS_PATH}/gcb_env.sh"
```

Comment:
not sure i follow.
why copy the file back?

Reply:
if the file does not exist on remote, copy from local